phpcs.xml.dist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<ruleset name="Options Importer">
+	<description>Sniffs for the coding standards of the Options Importer plugin</description>
+
+	<!--
+	Pass some flags to PHPCS:
+	 p flag: Show progress of the run.
+	 s flag: Show sniff codes in all reports.
+	-->
+	<arg value="ps" />
+
+	<!-- Only check the PHP files. -->
+	<arg name="extensions" value="php" />
+
+	<!-- Check all files in this directory and the directories below it. -->
+	<file>.</file>
+
+	<!-- Exclude a few directories and autogenerated files. -->
+	<exclude-pattern>*/vendor/</exclude-pattern>
+
+	<rule ref="WordPress"></rule>
```

Comment:
Very very minor, but these empty elements can be simplified to self-closing elements:  e.g. `<rule ref="..."/>`

Also, for all of the self-closing elements incl. `args` above), then no space is needed before the `/>` (if you normalize with XMLlint, it will remove these extra spaces) - this is XML, not XML-ized HTML :-)